### PR TITLE
feat(FDS-15): Form P1

### PR DESCRIPTION
**Pull request by brianespinosa**
Resolves FDS-15

Checklist:

- [x] make more evident that we can use `placeholders` in text inputs
- [x] make sure `save` button of `edit` module is of `type={'submit}`
- [x] more examples

### Snapshots

- due to additional module updates, some snapshots have changed showing icons
- data-testid attribute was removed from ActionBar, changing the snapshot

### Fixtures
Some fixtures in cosmos have changed

### Dependencies
A new `postinstall` script was added to build Icons

### CODE REVIEW

**Review by lordpixel — APPROVED** [review]

**Comment by brianespinosa** [comment]
@lordpixel note that this branch is currently building and deploying correctly. So we will need to see what is changed.

It looks like the Omni chat PR is also failing, but not sure if for the same reasons. I am about to pull develop into here to pick up our Epic tag checklist. That will give us another build here to validate if docs are still good in this branch or if something broke in develop too.

**Comment by brianespinosa** [comment]
Looks like there are target branch conflicts. Going to resolve those now before this branch gets borked.

File: packages/cascara/src/ui/Table/ActionBar.test.snap [hunk 1]
```diff
@@ -13,20 +13,22 @@ exports[`ActionBar component tree snapshot 1`] = `
     Espressive
   </span>
   <button
+    aria-label="new"
     class="ui basic button"
     data-testid="new"
```
Comment (brianespinosa) [inline packages/cascara/src/ui/Table/ActionBar.test.snap:18]: @lordpixel we should remove testid from these.

**Review by brianespinosa — COMMENTED** [review]
It looks like we are getting pretty close. Just some test-id things that should come out that I suspect are actually not being used in the tests anyway.

File: packages/cascara/src/ui/Form/Form.js [hunk 2]
```diff
@@ -1,12 +1,206 @@
...
+
+// separates the props into two groups:
+//
+// - myProps, all props defined in `propTypes`
+// - otherProps, the rest of the props
+//
+// @param {Object} props, the props object to filter
+// @param {Object} propTypes, a prop-types definition object
+// @returns {object}
+const filterProps = (props, propTypes) => {
```
Comment (lordpixel) [inline packages/cascara/src/ui/Form/Form.js:72]: change this to `unhandledProps`

**Comment by brianespinosa** [comment]
@lordpixel changing this PR out of draft state and ready to merge. I am marking it to auto merge as soon as you review and approve. Then we should get an alpha release deployed we can test with.

**Review by lordpixel — APPROVED** [review]

**Pull request merged**